extension.html

```diff
@@ -152,6 +152,11 @@ <h2>Extensions to the <code>PointerEvent</code> interface</h2>
 The events in the sequence will have increasing <a href="https://dom.spec.whatwg.org/#dom-event-timestamp"><code>timeStamps</code></a> ([[!WHATWG-DOM]]).
 So the first event will have the smallest <code>timeStamp</code> which should still be greater than the last event in the coalesced events.</p>
               </dd>
+
+              <p>
+                A <code>PointerEvent</code>'s <code> get target </code> algorithm, returns the associate dispatched event's target if it is a non-dispatch event (not being dispatched to a target, i.e. coalesced events
```

Comment:
This should link to some kind of associated event slot right? And presumably you'd then be able to branch on that for the return value, without having to be hand-wavy about it.

Reply:
I added the link to PointerEvent interface (@NavidZ could you please help me take a look if I did it correctly)
I also added the link to "dom#get-target" (which wasn't exist yet). I'll create the PR to dom once this approach get approved and try to merge these two at the same time.